ember-core/shared/src/test/scala/org/http4s/ember/core/ParserSuite.scala

```diff
@@ -254,7 +254,7 @@ class ParsingSuite extends Http4sSuite {
     } yield req1._1.method == Method.GET && req2._1.method == Method.GET).assert
   }
 
-  test("Parser.Response.parser should handle a chunked response") {
+  test(" should handle a chunked response") {
```

Comment:
I think we can revert this and the next two test name changes?
Most of the tests in the file still use the `Parser.Response.parser` prefix

Reply:
good catch, IntelliJ may have done this, will restore the names.